common/src/main/java/com/wynntils/screens/WynntilsQuestBookScreen.java

```diff
@@ -183,17 +188,13 @@ public boolean charTyped(char codePoint, int modifiers) {
 
     @Override
     public boolean mouseScrolled(double mouseX, double mouseY, double delta) {
-        if (delta < 0) {
-            setCurrentPage(getCurrentPage() + 1);
-        } else if (delta > 0) {
-            setCurrentPage(getCurrentPage() - 1);
-        }
+        setCurrentPage(getCurrentPage() + (delta > 0 ? -1 : 1));
```

Comment:
Just a reflection. I noticed aBackfish's PR on legacy, where there was a ScrollDirection.UP/DOWN selection. Maybe we should consider having a general function that converts a mouse scrolled delta to a direction, and that can be affected by settings. (Is there not already an inverted scroll setting in Vanilla, btw?)

Not saying it should be part of this PR, just ranting a bit. :)

Reply:
Yeah, if there is not one, we could have one.